oag/ja-jp/content/topics/access-gateway/trouble-shooting-guide.htm

```diff
@@ -369,10 +369,10 @@ <h2><a name="Jump_to_top_↑"></a>どのような問題が発生しましたか
                                                     </ul>
                                                     <h2 id="how-to-monitor-logs"><a name="How"></a>ログを監視するには</h2>
                                                     <div class="noteOkta">
-                                                        <p class="noteContent">ログは、ダウンロードすることも、ログサーバーに転送することもできます。詳細については、<a href="task-admin-logging.htm" class="MCXref xref">「ロギングの管理」</a>をご参照ください。</p>
+                                                        <p class="noteContent">ログは、ダウンロードすることも、ログサーバーに転送することもできます。詳細については、「<a href="task-admin-logging.htm" class="MCXref xref">ロギングの管理</a>」を参照してください。</p>
                                                     </div>
                                                     <ol>
-                                                        <li value="1"><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway 管理者コンソール</span>にログインします。デフォルトの資格情報は次のとおりです。<ul><li>ユーザー名： oag-mgmt</li><li>パスワード：OktaMgmt@123</li></ul></li>
+                                                        <li value="1"><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway管理コンソール</span>にログインします。デフォルトの資格情報は次のとおりです。<ul><li>ユーザー名： oag-mgmt</li><li>パスワード：OktaMgmt@123</li></ul></li>
```

Comment:
removes white space between en and ja part - is this intentional?

Reply:
The space is not necessary in a JA context